athenahealth/documents_test.go

```diff
@@ -88,6 +88,53 @@ func TestHTTPClient_AddDocument(t *testing.T) {
 	assert.NoError(err)
 }
 
+func TestHTTPClient_AddClinicalDocument(t *testing.T) {
+	assert := assert.New(t)
+
+	attachmentContents := []byte("test attachment contents")
+	autoclose := "true"
+	deptID := 2
+	documentSubclass := "CLINICALDOCUMENT"
+	internalNote := "test internal note"
+	providerID := 3
+	documentTypeId := 4
+
+	h := func(w http.ResponseWriter, r *http.Request) {
+		assert.NoError(r.ParseForm())
+
+		assert.Contains(r.URL.Path, "/patients/123/documents/clinicaldocument")
```

Comment:
```suggestion
		assert.Equal(r.URL.Path, "/patients/123/documents/clinicaldocument")
```

I believe the root path is appended later so you can equal assert here